site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -80,7 +123,8 @@ gcloud compute firewall-rules create game-server-firewall \
 
 ## Creating the cluster
 
-A [cluster][cluster] consists of at least one *control plane* machine and multiple worker machines called *nodes*. In Google Kubernetes Engine, nodes are [Compute Engine virtual machine][vms] instances that run the Kubernetes processes necessary to make them part of the cluster.
+
+### Create a Standard mode cluster for Agones
```

Comment:
If this is `###`, then let's move the "(Optional)" headers to `####` so the right hand side headers will come in under Standard as well.